File: Ejercicios de Semana Santa/ejercicios de semana santa php/Four php/Library/LicenciaB.php
<?php
require_once 'Licencia.php';

class LicenciaB extends Licencia {
    public function __construct() {
        $this->costo = 1300000;
        $this->duracion = 9;
    }

    public function mostrarServicios() {
        echo "Licencia B: vehículos particulares.\n";
    }
}
?>

File: Prueba y error/ejercicios de semana santa java/four java/Library/LicenciaB.java
package Library;

import java.text.DecimalFormat;

public class LicenciaB extends Licencia {
    public LicenciaB() {
        this.tipo = "LicenciaB (Vehículos particulares)";
        this.duracion = 24;  
        this.costo = 410000;  
    }

    @Override
    public String detallesLicencia() {
        // decimales . y , al valor de la conosola
        DecimalFormat formato = new DecimalFormat("#,###.00");
        String costoFormateado = formato.format(costo);
        return "Tipo de Licencia: " + tipo + "\nDuración: " + duracion + " meses\nCosto: $" + costoFormateado + " COP";
    }
}